beacon-node-oapi.yaml

```diff
@@ -96,6 +98,15 @@ paths:
   /eth/v1/debug/beacon/heads:
     $ref: './apis/debug/heads.yaml'
 
+  /eth/v1/lightclient/committee_updates:
```

Comment:
The API has historically defined `committee` as a beacon committee, and `sync_committee` as a sync committee, so I believe you want the latter here.

Reply:
Even if it's scoped in the lightclient namespace is there a risk of confusion?

Reply:
I support using `sync_committee` for clarity here.